paasta_tools/autoscaling/autoscaling_cluster_lib.py

```diff
@@ -326,8 +326,6 @@ def check_expected_slaves(
             if float(current_instances) / expected_instances < (1.00 - MISSING_SLAVE_PANIC_THRESHOLD):
```

Comment:
I think the autoscaler only refuses to scale when (current/expected < 1-missing_slave_panic_threshold) in scaling up. It can continue in all other cases, including (current/expected < 1) and is_scaling_down. 

When the autoscaler detects (current<expected), it is an opportunity to emit this data to signalfx.

Reply:
@stug It would nice if you can add this to paasta tools. We can have paasta find-orphans or something. This way, we(operators) can use it and see the instances that not part of the cluster.  You just need to lookup instances with Paasta IAM role to find the aws instances query mesos-master /slaves API to find the agents registered, and print the delta between the two. We can also have an option to emit metrics to Signalfx using '--emit-metrics' flag.

Reply:
@sagar8192 I feel like that's a little out of scope for this work...let's talk offline about whether that's work we want to prioritize or not.

Will add code to emit to signalfx for now.